### PR TITLE
refactor(aliases): rename node ID references to node alias for clarity

### PR DESCRIPTION
**Pull request by instamenta**
## Description

1. Created new aliases for:
  * pod name
  *  node alias 
  *  node id

2.  Renamed const values of the flags for internal use to reflect their actual usage and value:
  * nodeIDs --> nodeAliasesUnparsed 
  * nodeID --> nodeAlias

3. Renamed all references appropriately.

### Related Issues

* Closes # [527](https://github.com/hashgraph/solo/issues/527)

### CODE REVIEW

**Review by JeffreyDallas — DISMISSED** [review]

**Comment by JeffreyDallas** [comment]
I would appreicate we have a shorter branch name, since my computer screen is not wide enough :-)

File: src/core/templates.mjs [hunk 1]
```diff
@@ -224,50 +232,58 @@ export class Templates {
...
+  static renderFullyQualifiedNetworkSvcName (namespace, nodeAlias) {
+    return `${Templates.renderNetworkSvcName(nodeAlias)}.${namespace}.svc.cluster.local`
   }
 
   /**
    * @param {string} svcName
-   * @returns {string}
+   * @returns {NodeAlias}
    */
   static nodeIdFromFullyQualifiedNetworkSvcName (svcName) {
```
Comment (jeromy-cannon) [inline src/core/templates.mjs:255]:
```suggestion
  static nodeAliasFromFullyQualifiedNetworkSvcName (svcName) {
```

File: src/commands/flags.mjs
Comment (jeromy-cannon) [inline src/commands/flags.mjs:1]: rename `debugNodeId` to `debugNodeAlias`

File: src/commands/prompts.mjs
Comment (jeromy-cannon) [inline src/commands/prompts.mjs:1]: rename `promptNodeIds` to `promptNodeAliases`

File: src/commands/prompts.mjs [hunk 2]
```diff
@@ -383,10 +383,10 @@ export async function promptAmount (task, input) {
 
 export async function promptNewNodeId (task, input) {
```
Comment (jeromy-cannon) [inline src/commands/prompts.mjs:384]:
```suggestion
export async function promptNewNodeAlias (task, input) {
```

File: src/commands/node.mjs [hunk 3]
```diff
@@ -388,7 +389,7 @@ export class NodeCommand extends BaseCommand {
       // Create the transaction
       const transaction = await new AccountUpdateTransaction()
         .setAccountId(accountId)
-        .setStakedNodeId(Templates.nodeNumberFromNodeId(nodeId) - 1)
+        .setStakedNodeId(Templates.nodeNumberFromNodeAlias(nodeAlias) - 1)
```
Comment (jeromy-cannon) [inline src/commands/node.mjs:392]:
```suggestion
        .setStakedNodeId(Templates.nodeIdFromNodeAlias(nodeAlias) - 1)
```

File: src/commands/node.mjs
Comment (jeromy-cannon) [inline src/commands/node.mjs:1]: from:
```
          const lastNodeNumberMatch = lastNodeName.match(/\d+$/)
          if (lastNodeNumberMatch.length) {
            const incremented = parseInt(lastNodeNumberMatch[0]) + 1
            lastNodeName = lastNodeName.replace(/\d+$/, incremented.toString())
          }
```
```
          const lastNodeIdMatch = lastNodeAlias.match(/\d+$/)
          if (lastNodeIdMatch.length) {
            const incremented = parseInt(lastNodeIdMatch[0]) + 1
            lastNodeAlias = lastNodeAlias.replace(/\d+$/, incremented.toString())
          }
```

File: src/commands/node.mjs [hunk 4]
```diff
@@ -2082,8 +2086,8 @@ export class NodeCommand extends BaseCommand {
         task:
             async (ctx) => {
               const config = /** @type {NodeAddConfigClass} **/ ctx.config
-              const newNodeFullyQualifiedPodName = Templates.renderNetworkPodName(config.nodeId)
-              const nodeNumber = Templates.nodeNumberFromNodeId(config.nodeId)
+              const newNodeFullyQualifiedPodName = Templates.renderNetworkPodName(config.nodeAlias)
+              const nodeNumber = Templates.nodeNumberFromNodeAlias(config.nodeAlias)
               const savedStateDir = (config.lastStateZipPath.match(/\/(\d+)\.zip$/))[1]
               const savedStatePath = `${constants.HEDERA_HAPI_PATH}/data/saved/com.hedera.services.ServicesMain/${nodeNumber}/123/${savedStateDir}`
```
Comment (jeromy-cannon) [inline src/commands/node.mjs:2092]:
```suggestion
              const savedStatePath = `${constants.HEDERA_HAPI_PATH}/data/saved/com.hedera.services.ServicesMain/${nodeId}/123/${savedStateDir}`
```

File: src/commands/node.mjs [hunk 5]
```diff
@@ -2082,8 +2086,8 @@ export class NodeCommand extends BaseCommand {
         task:
             async (ctx) => {
               const config = /** @type {NodeAddConfigClass} **/ ctx.config
-              const newNodeFullyQualifiedPodName = Templates.renderNetworkPodName(config.nodeId)
-              const nodeNumber = Templates.nodeNumberFromNodeId(config.nodeId)
+              const newNodeFullyQualifiedPodName = Templates.renderNetworkPodName(config.nodeAlias)
+              const nodeNumber = Templates.nodeNumberFromNodeAlias(config.nodeAlias)
```
Comment (jeromy-cannon) [inline src/commands/node.mjs:2090]:
```suggestion
              const nodeId = Templates.nodeIdFromNodeAlias(config.nodeAlias)
```

File: test/test_util.js
Comment (jeromy-cannon) [inline test/test_util.js:1]: from:
```
export async function getNodeIdsPrivateKeysHash (networkNodeServicesMap, namespace, k8, destDir) {
```
to
```
export async function getNodeAliasPrivateKeysHash (networkNodeServicesMap, namespace, k8, destDir) {
```

Comment (jeromy-cannon) [reply test/test_util.js:1]: and update references inside of this method.

**Review by jeromy-cannon — CHANGES_REQUESTED** [review]
update `network_node_services.mjs`.`nodeName` to `nodeAlias`

**Comment by jeromy-cannon** [comment]
Update references of `nodeName` to `nodeAlias`

File: test/test_util.js
Comment (JeffreyDallas) [reply test/test_util.js:1]: Done

File: src/commands/node.mjs (hunk 4 above)
Comment (JeffreyDallas) [reply src/commands/node.mjs]: Done

File: src/commands/node.mjs
Comment (JeffreyDallas) [reply src/commands/node.mjs:1]: Updated

File: src/commands/node.mjs (hunk 3 above)
Comment (JeffreyDallas) [reply src/commands/node.mjs]: Updated

File: src/commands/prompts.mjs (hunk 2 above)
Comment (JeffreyDallas) [reply src/commands/prompts.mjs]: Updated

File: src/commands/prompts.mjs
Comment (JeffreyDallas) [reply src/commands/prompts.mjs:1]: Updated

File: src/commands/flags.mjs
Comment (JeffreyDallas) [reply src/commands/flags.mjs:1]: Updated

File: src/core/templates.mjs (hunk 1 above)
Comment (JeffreyDallas) [reply src/core/templates.mjs]: Updated

**Review by jeromy-cannon — APPROVED** [review]

**Pull request merged**